src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball() {
+        this.color = ColorSupplier.getRandomColor();
+        this.number = getRandomNumber();
+    }
```

Comment:
```suggestion
    public Ball(String color, int number) {
        this.color = color;
        this.number = number;
    }
```
we shouldn't hardcode values in the constructor